### PR TITLE
feat(godeltaprof): service info mapping injection

### PR DESCRIPTION
**Pull request by korniltsev**

### CODE REVIEW

File: godeltaprof/svcinfo/svcinfo.go [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+import (
+	"os"
+	"runtime"
+	"runtime/debug"
+	"sync"
+)
+
+const (
+	LabelNameServiceRepository = "service_repository"
```
Comment (cyriltovena) [inline godeltaprof/svcinfo/svcinfo.go:11]: I don't think you need this.

Comment (korniltsev) [reply godeltaprof/svcinfo/svcinfo.go:11]: I was going to add both mapping and labels in one PR, but decided to separate

**Review by kolesnikovae — COMMENTED** [review]
It looks like only heap, mutex, and block profiles will have version info embedded into mapping.BuildID. Are we going to do same with CPU and goroutine profiles?

I'm wondering if we do really need/want to inject the version information into mappings:
 - Use of static labels is probably the most straight forward and convenient way for push mode.
 - In pull mode, we could implement a generic mechanism of metadata propagation through the HTTP response headers. Grafana Agent could extract the headers and inject them as series labels.

**Comment by korniltsev** [comment]
I like headers idea much more than the current PR mapping implementation. @cyriltovena wdyt ?

**Pull request closed without merging**